Upstream sets -Werror but is full of warnings which then block the build.
Disable it to get it compiling

Upstream issue: https://gitlab.com/accounts-sso/signon-plugin-oauth2/-/issues/15

diff --git a/common-project-config.pri b/common-project-config.pri
index e149d90..557dc4d 100644
--- a/common-project-config.pri
+++ b/common-project-config.pri
@@ -19,8 +19,6 @@ UI_SOURCES_DIR  = ui/src
 
 QMAKE_CXXFLAGS += -fno-exceptions \
     -fno-rtti
-# we don't like warnings...
-unix:QMAKE_CXXFLAGS += -Werror
 
 TOP_SRC_DIR     = $$PWD
 
